x-pack/plugins/enterprise_search/public/applications/shared/layout/side_nav.tsx

```diff
@@ -63,15 +63,17 @@ export const SideNav: React.FC<SideNavProps> = ({ product, children }) => {
 
 interface SideNavLinkProps {
   to: string;
+  shouldShowActiveForSubroutes?: boolean;
```

Comment:
After rereading a few times this var name is really growing on me! It's very clear what it means and I was able to figure it out without having to ask for a screenshot (which I almost did at first haha) 💯

Reply:
Haha. I wanted to, but it was late in the day and and I was trying to get out the door so I thought I'd roll the dice 🎲